client_generic/ContentDownloader/SheepGenerator.cpp

```diff
@@ -302,6 +302,13 @@ bool	SheepGenerator::getControlPoints( SheepUploader *uploader )
 																					1.0,
 																					1.0 );
 
+	// Convert "r3d0.sheepserver.net" to "v3d0.sheepserver.net" 
```

Comment:
first off i think the code is wrong since it does not check for *starts with* "r".
it just finds *any* "r".

but even if that is fixed dasvo will be right that this is a horrible change.
i am open to how to do it better.

dasvo, you might not be familiar with the problem, but the server has a bug that the logged-in (registered with the "r") URLs are broken and I have spent quite a bit of time trying to fix the server and failed.
so this hack of switching registered to unregistered URLs is my idea.
what else can we do?